server/src/main/java/io/druid/server/coordinator/helper/DruidCoordinatorBalancer.java

```diff
@@ -103,6 +103,7 @@ private void balanceTier(
       return;
     }
 
+    final List<ServerHolder> entireServerHolderList = Lists.newArrayList(servers);
```

Comment:
maybe rename these to "sourceServerHolderList" and "destinationServerHolderList"

Reply:
didn't use those exactly, but gave those and some other variables some better names